dluhc-component-library/src/components/timetablePage/ProgressTile.tsx

```diff
@@ -0,0 +1,15 @@
+interface ProgressTileProps {
+  progress: "notStarted" | "delayed" | "inProgress" | "finished";
```

Comment:
This type should be pulled out into a type file and reused in the Timetable component